src/gui/MainWindow.cpp

```diff
@@ -552,13 +555,20 @@ void MainWindow::finalize()
 								m_toolBar );
 	controllers_window->setShortcut( Qt::Key_F11 );
 
+	m_countDownClock = new LcdSpinBox(2, this, tr("Count Down Clock"));
+
+	m_countDownClock->setLabel( tr("SEC") );
+	m_countDownClock->setModel( m_countDownValue );
+	connect( m_countDownValue, SIGNAL(dataChanged()), this, SLOT( updateCountDownDuration() ) );
```

Comment:
Ahh, there's a small problem, I don't know why the `manualChange` wasn't fired during my testing.